orttraining/orttraining/python/training/_ortmodule_io.py

```diff
@@ -313,7 +350,9 @@ def parse_inputs_for_onnx_export(all_input_parameters, onnx_graph, inputs, kwarg
                       shape=input_shape,
                       require_grad_names=input_names_require_grad,
                       dynamic_axes=dynamic_axes,
-                      schema=schema)
+                      schema=schema,
+                      num_positionals=len(inputs),
+                      keyword_names=sorted(kwargs.keys()))
```

Comment:
Btw, I'm not 100% sure if the `sorted` from my example is necessary (or a good idea) :). It might be a better trade-off to rely on the natural order of Python 3.x dicts, and avoid sorting on each call.

Reply:
I had a TODO on the previous iteration for this exact question, but for this spot, I think the natural ordering is fine.